source/containers.tex

```diff
@@ -6338,7 +6338,7 @@
 \pnum
 \requires
 \tcode{is_assignable_v<mapped_type\&, M\&\&>} shall be \tcode{true}.
-\tcode{value_type} shall be \tcode{EmplaceConstructible} into \tcode{map}
+\tcode{value_type} shall be \tcode{Emplace\-Constructible} into \tcode{map}
```

Comment:
I think it may be because the text is inside a `\tcode`. We could probably also change that somehow. The `hyphenat` package does something similar.